css/styles.css

```diff
@@ -0,0 +1,111 @@
+header {
+  background-color: #1d3557;
+  position: relative;
+  z-index: 2;
+}
+
+.heading-container {
+  color: white;
+  display: flex;
+  text-align: center;
+  justify-content: center;
+  align-items: baseline;
+  position: relative;
+  z-index: 2;
+}
+
+.main-container {
+  position: relative;
+  margin-top: 5%;
+  z-index: 2;
+}
+
+body {
+  position: relative;
+  font-family: 'Montserrat', sans-serif;
+}
+
+h4 {
+  margin: 5px 0 0 0;
+}
+
+.background-pic {
+  display: block;
+  position: absolute;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  z-index: 1;
+  opacity: 0.25;
+}
```

Comment:
This is not the ideal way to set a background image. Are you doing this so that you can set the `src` to the image you get from your api? You can do that with `element.style.background = "url(" + theUrl + ")"` in your JS. That should also remove the need for all the z-indexes here.